src/main/java/org/restcomm/sdk/domain/CallStatus.java

```diff
@@ -0,0 +1,33 @@
+package org.restcomm.sdk.domain;
+
+/**
+ * A string representing the status of the call.
+ * <p>
+ * Possible values are queued, ringing, in-progress, completed, failed, busy and no-answer.
+ *
+ * @author Henrique Rosa (henrique.rosa@telestax.com) created on 21/05/2018
+ */
+public enum CallStatus {
```

Comment:
Same question as about CallDirection

Reply:
I do assertions based on call status for synthetic monitoring. It's better to rely on enum rather than hardcode string values everywhere.